include/intro_movie.h

```diff
@@ -0,0 +1,8 @@
+#ifndef POKEHEARTGOLD_INTRO_MOVIE_H
+#define POKEHEARTGOLD_INTRO_MOVIE_H
+
+#include "overlay_manager.h"
+
```

Comment:
this feels there should be more here, everything public facing should be here imo

Reply:
intro_movie.h is for symbols and types that are exposed to the rest of the game
intro_movie_internal.h is for symbols and types that should be forward-declared only within the intro_movie overlay sources
the headers are correct as is

Reply:
does that mean that the only exposed thing is struct?